pkg/mock/vault.go

```diff
@@ -0,0 +1,75 @@
+// +build !release
```

Comment:
I was actually thinking about using it, but was not sure if mockery is used in this project since i didn't see it in the other mocks.

I'll change it :smile: